site-book/README.md

```diff
@@ -0,0 +1,50 @@
+# Metron Site-Book documentation
+
+Metron's Site Book is an attempt at producing documentation that is
```

Comment:
Maybe I'm being a bit of a pedant, but I would put a \: at the end of this line.